include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
 
 private:
     /*!
-     * \brief Default constructor.
-     *
-     * \note This constructor is not implemented and should not be used.
-     */
-    IBImplicitStaggeredHierarchyIntegrator();
+	 * \brief A StaggeredStokesSolver that does nothing.
```

Comment:
We need to get info like u_problem_coef, nullspace info etc from INSHierarchyIntegrator, but we do not need to create any StokesSolver within ins hierarchy integrator. I don't think we need to change design, because to me provision for registering external stokes solver with ins integrator seems to be precisely for these situations...